src/components/user/PersonalProfileEditForm.js

```diff
@@ -44,6 +55,23 @@ export default function PersonalProfileEditForm() {
       });
     } else {
       try {
+        // If there's a profile picture, upload it first
+        if (formData.profilePicture) {
+          const formDataImage = new FormData();
+          formDataImage.append('image', formData.profilePicture);
```

Comment:
The backend is expecting that we send this file with the name `profile` not `"image"`

```suggestion
          formDataImage.append('profile', formData.profilePicture);
```

Here: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/311fd8c5230019e23d6933c28811a08f81566a89/src/controllers/imgurController.js#L35-L41

Reply:
thank you ❤️